README.md

```diff
@@ -1,12 +1,16 @@
-![Sylius](http://demo.sylius.org/assets/shop/img/logo.png)
+<h1 align="center">
+    <a href="http://sylius.com" target="_blank">
+        <img src="https://demo.sylius.com/assets/shop/img/logo.png" />
+    </a>
+</h1>
 
 [![License](https://img.shields.io/packagist/l/Sylius/Sylius.svg)](https://packagist.org/packages/sylius/sylius)
 [![Version](https://img.shields.io/packagist/vpre/Sylius/Sylius.svg)](https://packagist.org/packages/sylius/sylius)
-[![Build status on Linux](https://img.shields.io/travis/Sylius/Sylius/master.svg)](http://travis-ci.org/Sylius/Sylius)
+[![Build Status on Linux](https://travis-ci.org/Sylius/SyliusDemo.svg?branch=master)](https://travis-ci.org/Sylius/SyliusDemo)
 [![Scrutinizer Quality Score](https://img.shields.io/scrutinizer/g/Sylius/Sylius.svg)](https://scrutinizer-ci.com/g/Sylius/Sylius/)
 [![Total Downloads](https://poser.pugx.org/sylius/sylius/downloads)](https://packagist.org/packages/sylius/sylius)
 
-**This is Sylius Demo Edition repository deployed to demo.sylius.org.**
+**This is Sylius Demo Edition repository deployed to [demo.sylius.org](https://demo.sylius.com/).**
```

Comment:
One more thing, it could be `demo.sylius.com`, as well as `docs.sylius.com` below :)